exosphere-shared/src/logger.spec.ls

```diff
@@ -0,0 +1,54 @@
+require! {
+  'chai': {expect}
+  'chalk' : {yellow}
+  './logger': Logger
+}
+
+describe 'Logger', ->
+
+  describe '_parse-line', ->
+    logger = new Logger [role]
+    role = 'exo-run'
+
+    specify 'should parse non-service log message correctly' ->
+      line = 'Attaching to exocom0.21.8, web'
+      logger._parse-line role, line, (left, right) ->
+        expect(left).to.eql(role)
+        expect(right).to.eql(line)
```

Comment:
Can we update `_parse-line` to return an object? I'm envisioning the interface to look like the following. I think thats a better way to return multiple arguments then passing a callback.
```livescript
result = logger._parse-line role, line
expect(result.left).to.eql #...
expect(result.right).to.eql #...
```